### ISSUE TITLE
VS Code TS Server fatal error: Maximum call stack size exceeded in URL.onParseComplete

### ISSUE THREAD

**Issue by mfellner** [justjavac/typescript-deno-plugin#39]
**Describe the bug**

Getting the following error in VS Code: `The TypeScript language service died 5 times right after it got started. The service will not be restarted.` and `TS Server fatal error:  Maximum call stack size exceeded`.

**To Reproduce**

1. Create mod.ts
2. Add this line:
```ts
import * as pkg from 'https://cdn.pika.dev/preact-render-to-string@5.1.8';
```
3. `The TypeScript language service died 5 times...`

With a different URL:

1. Create mod.ts
2. Add this line:
```ts
import * as pkg from 'https://cdn.pika.dev/preact-render-to-string@^5.1.8';
```
3. `Cannot find module 'https://cdn.pika.dev/preact-render-to-string@^5.1.8'`

**Expected behavior**

Import should work.

**Stack traces**

```
RangeError: Maximum call stack size exceeded
    at new URLSearchParams (internal/url.js:126:14)
    at URL.onParseComplete (internal/url.js:236:26)
    at parse (<anonymous>)
    at new URL (internal/url.js:319:5)
    at Object.resolve (/****/.vscode/extensions/justjavac.vscode-deno-1.28.0/node_modules/typescript-deno-plugin/out/module_resolver/remote_module_resolver.js:14:21)
    at Object.resolve (/****/.vscode/extensions/justjavac.vscode-deno-1.28.0/node_modules/typescript-deno-plugin/out/module_resolver/universal_module_resolver.js:11:66)
    at Object.resolve 
```

### Versions

**OS:** macOS 10.15.4
**VS Code version:** 1.45.0
**VS Code TypeScript version:** 3.8.3
**Deno version:** 1.0.0
**Deno TS version:** 3.9.2

**Comment by justjavac** [comment]
fixed by be90458d936de5a37ac1d68f0cf08feacfd157ef

**Comment by IgorSzymanski** [comment]
Now it totally ignores type definitions and everything is either inferred as `{}` or `any`.
All the types from .ts modules served by deno.land still work fine, which is kind of obvious.

[ReactDOM](https://cdn.pika.dev/react-dom@^16.13.1)
![Screenshot from 2020-05-19 06-18-37](https://user-images.githubusercontent.com/12682069/82284283-f56f1d00-9998-11ea-88a8-b819a6818094.png)

[React](https://cdn.pika.dev/react@^16.13.1)
![Screenshot from 2020-05-19 06-19-17](https://user-images.githubusercontent.com/12682069/82284287-f6a04a00-9998-11ea-9434-385710dca6c9.png)

[http](https://deno.land/std/http/mod.ts)
![Screenshot from 2020-05-19 06-19-34](https://user-images.githubusercontent.com/12682069/82284289-f738e080-9998-11ea-8d44-4cc02ece6f1d.png)


```ts
// server.tsx
import React, { Component } from 'https://cdn.pika.dev/react@^16.13.1'
import ReactDOM from 'https://cdn.pika.dev/react-dom@^16.13.1'
import { serve } from 'https://deno.land/std/http/mod.ts'

console.log(serve)
console.log(ReactDOM)
console.log(React)
console.log(Component)
```
